### ISSUE TITLE
Closing repo?

### ISSUE THREAD

**Issue by joramkruijer** [Bitblade/GyrinusDeur#4]
Due to developments on the side of a board member of Gyrinus (with an obvious desert-quantity load of sand in her vagina), should we:
- Close this repo or
- Continue anyway, be it clandestine or as open as before
Also: more hay? I think more hay works. More hay.

**Comment by Bitblade** [comment]
Agree, this clearly calls for more hay. However, we need a more efficient hay delivery mechanism.

Obviously the easiest way would be to open the door, but somehow I imagine @Penningmeester will not allow us to do that. 

Instead, I think we should go with a compressed air delivery system. Shall I bring my mobile compressor next time?
